bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -5220,16 +5658,30 @@ msgid "Select a data interface"
 msgstr "Seleccione una interfaz de datos"
 
 #: bika/lims/content/analysisservice.py:208
-msgid "Select a default preservation for this analysis service. If the preservation depends on the sample type combination, specify a preservation per sample type in the table below"
-msgstr "Seleccione un método de conservación para el servicio de análisis. Si el método de preservación depende de la combinación de tipos de muestra, especifique un método de preservación por tipo de muestra de la tabla siguiente."
+msgid ""
+"Select a default preservation for this analysis service. If the preservation "
+"depends on the sample type combination, specify a preservation per sample "
+"type in the table below"
+msgstr ""
+"Seleccione un método de conservación para el servicio de análisis. Si el "
```

Comment:
@xispa maybe use one of the words _preservación_ or _conservación_ but not both? It may lead to confusion.